modules/govuk/manifests/node/s_mapit.pp

```diff
@@ -2,6 +2,7 @@
 #
 class govuk::node::s_mapit inherits govuk::node::s_base {
 
+  include govuk_python3
```

Comment:
All the changes LGTM, except perhaps this one - why is this not `govuk_python` rather than `govuk_python3`?

Reply:
I updated the PR description to include:
```
python3.6 is installed on mapit instances using govuk-python3 repo
rather than mistakenly the govuk-python repo. This is required since
installation of Python 2.7.14 is via govuk-python repo and we can't
have 2 co-existing versions of the same apt package.
```